app/services/verification_errors_report.rb

```diff
@@ -0,0 +1,20 @@
+class VerificationErrorsReport
+  def self.call(service_provider, since)
+    report_sql = <<~SQL
+      SELECT agency_identities.uuid,welcome_view_at,document_capture_submit_at,ssn_view_at,
+        verify_submit_at,ssn_view_at,verify_phone_submit_at,encrypt_view_at,enter_info_view_at
```

Comment:
let's break these out into rows for easier skimming?
```suggestion
      SELECT
        agency_identities.uuid,
        welcome_view_at,
        document_capture_submit_at,
        ssn_view_at,
        verify_submit_at,
        ssn_view_at,
        verify_phone_submit_at,
        encrypt_view_at,
        enter_info_view_at
```